__tests__/tests.test.js

```diff
@@ -112,3 +112,49 @@ describe("/api/articles", () => {
       });
   });
 });
+describe.only("/api/articles/:article_id/comments", () => {
+  test("GET 200: responds with a 200 status code and comment when only one comment for that article", () => {
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body = body[0];
```

Comment:
I would probably rename this variable. Not sure this test is needed - its similar to the one below?